dive_core/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@
 project(dive_core)
 
 set(CMAKE_CXX_STANDARD 20)
-
+include(CheckFunctionExists)
```

Comment:
do we use CHECK_FUNCTION_EXISTS_ADD_LIBRARIES in the proj somewhere? i couldn't find it. or is this for something else?

Reply:
oh, i see. good to know. thanks